maps/randomvaults/objects.dm

```diff
@@ -710,15 +710,15 @@
 	if(surplus() < active_power_usage)
 		powered = 0
 
-	if(powered && stat & NOPOWER)
+	if(powered && stat & (NOPOWER))
 		stat &= ~NOPOWER
 		update_icon()
 	else if (!powered && !(stat & NOPOWER))
 		stat |= NOPOWER
 		update_icon()
 
 /obj/machinery/power/magtape_deck/update_icon()
-	if(stat & (BROKEN|NOPOWER))
+	if(stat & (BROKEN|NOPOWER|FORCEDISABLE))
```

Comment:
I realize this probably was caused by a regex operation but you can take all this out, I don't think the station AI should have control over a syndicate vault clue machine, even when malfunctioning.